samples/HelloWorld/HelloWorld.al

```diff
@@ -11,6 +11,6 @@ codeunit 70051001 HelloWorld
     var
         HelloText : Codeunit GreetingsManagement;
     begin
-        Message(HelloText.GetRandomGreeting() + ', ' + Rec.Name);
+        Message('%1, %2',HelloText.GetRandomGreeting(),Rec.Name);
```

Comment:
I do not agree with the variable names in texts. If there is no error when renaming variable (and there will not be...), and in case when you separate language from the source code, it will be hard or impossible to maintain...